blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineImpl.java

```diff
@@ -179,4 +179,9 @@ public Resource resolve(Item context, Reachable parent, Item target) {
 
     }
 
+    @Exported(inline = true)
+    @Navigable
+    public Container<Resource> getActivities() {
+        return Containers.fromResource(getLink(),Lists.newArrayList(Iterators.concat(getQueue().iterator(), getRuns().iterator())));
```

Comment:
so will this concatenation preserve pagination? ie it won't force a full load accidentally?